overrides/offline.yaml

```diff
@@ -1,4 +1,4 @@
-os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_major_version }}_x86_64.tar.gz"
+os_packages_local_bundle_file: "{{ playbook_dir }}/../artifacts/{{ kubernetes_version }}_{{ ansible_distribution|lower }}_{{ ansible_distribution_version }}_x86_64.tar.gz"
```

Comment:
Need a similar change in https://github.com/mesosphere/konvoy-image-builder/blob/964bfb20aa2801cbcafcde7efa9148449b1cf6bb/overrides/offline-fips.yaml#L2